tests/test_integrity.py

```diff
@@ -423,6 +423,16 @@ def test_modeled_parameters(db):
         print(t)
     assert len(t) == 0
 
+    t = (
+        db.query(db.ModeledParameters)
+        .filter(db.ModeledParameters.c.model.is_(None))
+        .astropy()
+    )
+    if len(t) > 0:
+        print("\nEntries found without a model")
+        print(t)
+    assert len(t) == 705
+
```

Comment:
This is not a data "integrity" tests like the other ones that are here. Make a new `test_data_modeled_params.py` file for this test.  It's totally fine for modeled parameter entries to not have a model.